locales/templates/app.pot

```diff
@@ -531,15 +531,27 @@ msgctxt "firstrun_next_button"
 msgid "Next"
 msgstr ""
 
-#. The welcome titles in Firefox Lite 2.5
+#. !NOT USE! The welcome titles in Firefox Lite 2.5
 msgctxt "firstrun_fxlite_2.5_title_1"
 msgid "Hello!"
 msgstr ""
 
+#. !NOT USE! The welcome titles in Firefox Lite 2.5 for asking users to choose
+#. a main purpose of downloading Firefox Lite. The %1$s is Firefox Lite.
```

Comment:
Probably better not to name the product in case it changes in the future

Reply:
Makes sense to me.
I didn't notice that before, so there are tons of "The %1$s is Firefox Lite" spreading in the string file......Would you think it's still fine to refine them all at once when we rename our product?

Reply:
I think we can refine them at once in an upcoming PR just to say something generic (so: "(...)will be replaced by product name", for example). That way you won't ever to update it ever again! :)